docs/headless-cms/docs/03-01-headless-cms-in-console.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Headless CMS in Console
+type: Details
+---
+
+Headless CMS in Kyma is a separate component and does not provide templates or a standalone UI to display content from CMS. Still, some of its assets are shown in the Console UI in the Docs view (generic and component-related documentation) and the Service Catalog view (API specifications).
+
+You add a new documentation topic to the Console UI by creating a CR, while the [Asset Store](#asset-store-overview) handles the rest. You define the location, grouping, and order of the documents through these DocsTopic and ClusterDocsTopic labels:
```

Comment:
I don't think `#asset-store-overview` is a valid link

Reply:
```suggestion
You add a new documentation topic to the Console UI by creating a CR, while the [Asset Store](#asset-store-overview) handles the rest. You define the location, grouping, and order of the documents through these DocsTopic and ClusterDocsTopic labels. This is a typical filtering operation that is not part of custom resources specification but instead we introduced below labels convetion:
```

Reply:
Changed it a bit, pls take a look ;)

Reply:
I don't see any change really